### ISSUE TITLE
Incorrect System.Collections.Immutable.xml generated.

### ISSUE THREAD

**Issue by NitishMittalHon** [dotnet/corefx#29600]
I ended up opening System.Collections.Immutable.xml while trying to chase the reason behind Swagger failing to generate API docs.
Notice the \<br\> element in the below XML which I copied from System.Collections.Immutable.1.4.0. **\<br\> is not having a closing \</br\>.**
```xml
<member name="M:System.Collections.Immutable.ImmutableArray`1.CastArray``1">
<summary>
<p sourcefile="System.Collections.Immutable.yml" sourcestartlinenumber="1" sourceendlinenumber="2">Initializes a new instance of the <xref href="System.Collections.Immutable.ImmutableArray`1"></xref> struct by casting the underlying             array to an array of type <typeparameter name="TOther"><code data-dev-comment-type="paramref">TOther</code></typeparameter>
<br>.
</p>
</summary>
      <typeparam name="TOther"></typeparam>
      <returns></returns>
      <exception cref="T:System.InvalidCastException">Thrown if the cast is illegal.</exception>
</member>
```
For now I have ignored all the System.* files to be considered for swagger. I thought it would still be good to fix it.
Thanks.

**Comment by karelz** [comment]
@weshaggard do you know where the XML file comes from? Is it generated from docs?

**Comment by weshaggard** [comment]
Yes this comes from the docs team (cc @mairaw) but I'm almost certain this has been fixed in the doc files since the 1.4.0 package.
